config.w32

```diff
@@ -213,6 +213,7 @@ if (PHP_MONGODB != "no") {
     mongoc_opts.MONGOC_ENABLE_CRYPTO_LIBCRYPTO = 1;
     mongoc_opts.MONGOC_ENABLE_SSL = 1;
     mongoc_opts.MONGOC_ENABLE_CRYPTO = 1;
+    mongoc_opts.MONGOC_ENABLE_MONGODB_AWS_AUTH = 1;
```

Comment:
Noting that this is always safe to enable, since we require TLS on Windows.

Reply:
Ah, good to know. I'll keep it within the conditional to avoid nasty surprises down the road if we ever loosen the requirement.

Reply:
For reference, the dependency is [here](https://github.com/mongodb/mongo-php-driver/blob/aceebe78004910d1b85f8577c6672a846a5fd68c/config.w32#L59).